arkouda/sparrayclass.py

```diff
@@ -94,6 +100,56 @@ def __str__(self):  # This won't work out of the box for sparrays need to add th
     #     print("Called repr")
     #     return generic_msg(cmd="repr", args={"array": self, "printThresh": sparrayIterThresh})
 
+    """
+    Converts the sparse matrix to a list of 3 pdarrays (rows, cols, vals)
+    Returns
+    -------
+    List[ak.pdarray]
+        A list of 3 pdarrays which contain the row indices, the column indices,
+        and the values at the respective indices within the sparse matrix.
+
+    Raises
+    ------
+    RuntimeError
+        Raised if there is a server-side error thrown, if the pdarray size
+        exceeds the built-in client.maxTransferBytes size limit, or if the bytes
```

Comment:
Is this true? Afaik, the `maxTransferBytes` limit should only apply to converting to/from numpy arrays because the data have to be sent to/from the client. In this case, aren't you only creating three new array symbols on the server side and then sending their handles back to the client?

Reply:
Ah, that's a copy paste error, nice catch.